Core/Core/Extensions/UIViewControllerExtensions.swift

```diff
@@ -97,6 +97,10 @@ extension UIViewController {
         }
         child.didMove(toParent: self)
     }
+
+    public func dismiss(_ viewControllerToDismiss: UIViewController, animated flag: Bool, completion: (() -> Void)?) {
+        viewControllerToDismiss.dismiss(animated: flag, completion: completion)
+    }
```

Comment:
I don't understand why this extension is better than just calling the function.

Reply:
Oh yeah I don't need this anymore. It was mainly done so that I can call this from a presenter and test it. But it was smelly so I worked around it. I'll remove this.